oneapi/defaults/main.yaml

```diff
@@ -2,6 +2,7 @@
 
 # NSP_ONEAPI_base_toolkit_url
 NSP_ONEAPI_base_toolkit_components:
+  - "intel.oneapi.lin.dpcpp-cpp-compiler"
```

Comment:
This was already fixed in upstream by PR #2 , if you'd like to just cherry-pick/re-base on that commit.